ogr/ogrspatialreference.cpp

```diff
@@ -11998,11 +11998,39 @@ const int *OSRGetDataAxisToSRSAxisMapping(OGRSpatialReferenceH hSRS,
 /************************************************************************/
 
 /** \brief Set a custom data axis to CRS axis mapping.
+ *
+ * The number of elements of the mapping vector should be the number of axis
+ * of the CRS (as returned by GetAxesCount()) (although this method does not
+ * check that, beyond checking there are at least 2 elements, so that this
+ * method and setting the CRS can be done in any order).
+ * This is taken into account by OGRCoordinateTransformation to transform the
+ * order of ordinates to transform to the order expected by the CRS before
```

Comment:
+1, I think that a coordinate (the pair, or triplet) can have many coordinates (values along axes) but not many ordinates, because ordinate is the second axis that follows the abscissa.
So the history of the messy standards in GIS does not begin from OpenGIS but from Descartes.